Libraries/Cordio/platform/targets/maxim/max32655/sources/pal_uart.c

```diff
@@ -92,7 +94,12 @@ void UART_CommonHandler(mxc_uart_regs_t *uart)
   {
     const uint8_t uartIdx = MXC_UART_GET_IDX(uart);
     
-    if( uartIdx == CONSOLE_UART || uartIdx == HCI_UART)
+    if(uartIdx == CONSOLE_UART)
```

Comment:
Does this solve the problem we're having? I haven't seen RX errors in the testing that I've done.

Reply:
I have gotten a handful of overflow errors when it is automated.  You probably won't when manually doing it. It used to be an assertion that fired off, which happened 5-10% of runs. It is my best guess right now for the no action bound issue I see.

Reply:
I think we're better off preventing the overflow with slower writes from the automation.

Reply:
I will close it if you want, but we can't guarantee that the issue wont happen regardless.